free-1.md

```diff
@@ -10,41 +10,79 @@ _What need is met by your idea? Who are you helping? What is that person's
 experience like now? What would their experience be like if you could help 
 them?_
 
+Since Twitch Plays Pokemon began in 2014, there has been lots of interest
+in creating similar "crowdplay" games. However, it is an area primarily
+done by people willing to overcome the steep curve of learning how to interface
+with emulators and Twitch's API. A DSL could help people who would have previously
+been excluded from making a "Twitch Plays"
+
 
 ### Why a language?
 _Why is a DSL appropriate for your user(s)? How does it address the need?_
 
+A DSL is appropriate for multiple reasons. We are focusing on a specific domain,
```

Comment:
I wonder if it would be better to create an external DSL or an internal DSL - from what I understand, it seems like Python is currently used as an internal DSL to create Twitch Plays.